ee/codegen_integration/fixtures/simple_map_node/code/nodes/map_node/nodes/search_node.py

```diff
@@ -1,11 +1,11 @@
 from vellum import SearchFiltersRequest, SearchRequestOptionsRequest, SearchResultMergingRequest, SearchWeightsRequest
 from vellum.workflows.nodes.displayable import SearchNode as BaseSearchNode
 
-from ....nodes.map_node.inputs import Inputs
+from ..inputs import Inputs
 
 
 class SearchNode(BaseSearchNode):
-    query = Inputs.items
+    query = Inputs.item
```

Comment:
The fixture was wrong and made for an invalid workflow. The query to the search should be the individual item. Update the ID below to reflect this.